helm/examples/azure/hpcc-azurefile/templates/_helpers.tpl

```diff
@@ -61,3 +61,82 @@ Create the name of the service account to use
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Add common mountOptions
+*/}}
+{{- define "hpcc-azurefile.addCommonMountOptions" }}
+mountOptions:
+- uid=10000 # uid of user 'hpcc'
+- gid=10001 # gid of group 'hpcc'
+- dir_mode=0700 # user read/write/execute
+- file_mode=0600 # user read/write
+{{- if .root.Values.common.shareName }}
+- mfsymlinks
+- nobrl
+{{- end -}}
+{{- end }}
+
+{{/*
+Create PersistentVolume
```

Comment:
trivial: as with helper function in helm/hpcc/templates/_helpers.tpl, it's good to document what each helper function expects, e.g. 
"Pass in dict with root"

Reply:
Added